src/Apps/Conversation/Components/ConversationCTA.tsx

```diff
@@ -14,13 +14,11 @@ import { ReviewOfferCTA } from "./ReviewOfferCTA"
 import { PurchaseOnInquiryButtonFragmentContainer } from "./PurchaseOnInquiryButton"
 import { MakeOfferOnInquiryButtonFragmentContainer } from "./MakeOfferOnInquiryButton"
 import { ConversationCTA_conversation$data } from "__generated__/ConversationCTA_conversation.graphql"
-import { useFeatureFlag } from "System/useFeatureFlag"
 import { themeGet } from "@styled-system/theme-get"
 
 interface ConversationCTAProps {
   conversation: ConversationCTA_conversation$data
-
-  openInquiryModal: ({ createsOfferOrder: boolean }) => void
+  openInquiryModal: (obj: { createsOfferOrder: boolean }) => void
```

Comment:
**question:** The `obj:` before the the curly braces is new in this side of the diff. Is that something added by a linter?

Reply:
nah this was an intentional change in response to a warning. the type declaration was incorrect before; it was declaring `createsOrderOffer` with type `any` and then aliasing it to the word `boolean`